cmd/authelia/main.go

```diff
@@ -120,7 +120,15 @@ func main() {
 		},
 	}
 
-	rootCmd.AddCommand(versionCmd, commands.HashPasswordCmd, commands.MigrateCmd)
+	hashPasswordCmd := commands.HashPasswordCmd
+	hashPasswordCmd.Flags().BoolP("sha512", "z", false, "use sha512 instead of argon2id")
+	hashPasswordCmd.Flags().IntP("times", "t", 3, "set the number of hashing rounds")
```

Comment:
Per our discussion I'd prefer this to be called `iterations` or `rounds`.

```suggestion
	hashPasswordCmd.Flags().IntP("iterations", "i", 3, "set the number of hashing rounds")
```

Reply:
Done!